apps/linode-marketplace-openlitespeed-django/roles/post/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+-
+# motd and creds gen
+- name: copy MOTD template to /etc/motd
+  template:
+    src: templates/motd.j2
+    dest: /etc/motd
+
+- name: writing openlitespeed creds into file
+  copy:
+    dest: /root/.credentials
+    mode: '0600'
+    content: |
+      All log in credentials and setup instructions can be found after logging in via SSH.
```

Comment:
We should put the creds only in the sudo user's home dir

Reply:
Done.